### PR TITLE
MSI: integration tests on travis

### PR DESCRIPTION
**Pull request by naydav**
Fix integration tests on Travis

### CODE REVIEW

File: app/code/Magento/Inventory/Setup/Operation/CreateSourceItemTable.php [hunk 1]
```diff
@@ -33,6 +33,7 @@ public function execute(SchemaSetupInterface $setup)
     private function createSourceItemTable(SchemaSetupInterface $setup): Table
     {
         $sourceItemTable = $setup->getTable(SourceItemResourceModel::TABLE_NAME_SOURCE_ITEM);
+        $sourceTable = $setup->getTable(SourceResourceModel::TABLE_NAME_SOURCE);
```
Comment (ishakhsuvarov) [inline app/code/Magento/Inventory/Setup/Operation/CreateSourceItemTable.php:36]: `$sourceTableName` would be more informative

File: app/code/Magento/InventoryApi/Test/_files/source_items.php [hunk 2]
```diff
@@ -61,7 +61,7 @@
 $resourceConnection = Bootstrap::getObjectManager()->get(ResourceConnection::class);
 /** @var \Magento\Framework\DB\Adapter\AdapterInterface $connection */
 $connection = $resourceConnection->getConnection();
-$connection->query('ALTER TABLE inventory_source_item AUTO_INCREMENT 1;');
+$connection->query('ALTER TABLE ' . $resourceConnection->getTableNAme('inventory_source_item') . ' AUTO_INCREMENT 1;');
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryApi/Test/_files/source_items.php]: `getTableNAme` seems to be a typo

Comment (naydav) [reply app/code/Magento/InventoryApi/Test/_files/source_items.php]: Fixed

**Review by larsroettig — COMMENTED** [review]
For me okay

**Pull request merged**